operator/cmd/main.go

```diff
@@ -86,7 +86,7 @@ func main() {
 		"If set, HTTP/2 will be enabled for the metrics and webhook servers")
 	flag.StringVar(&programFSAddr, "program-fs-addr", envOrDefault("PROGRAM_FS_ADDR", ":9090"),
 		"The address the static file server binds to.")
-	flag.StringVar(&programFSAdvAddr, "program-fs-adv-addr", envOrDefault("PROGRAM_FS_ADV_ADDR", ""),
+	flag.StringVar(&programFSAdvAddr, "program-fs-adv-addr", envOrDefault("PROGRAM_FS_ADV_ADDR", "localhost:9090"),
```

Comment:
Rationale: I was finding that the vscode launch configuration for the operator was failing because an advertised address is required.  This seems like a reasonable default.